src/main/java/com/google/firebase/auth/FirebaseUserManager.java

```diff
@@ -287,7 +295,7 @@ String getEmailActionLink(EmailLinkType type, String email,
 
   private <T> T sendRequest(
           String method, GenericUrl url,
-          @Nullable Object content, Class<T> clazz) throws FirebaseAuthException {
+          Object content, Class<T> clazz) throws FirebaseAuthException {
```

Comment:
Why was the annotation removed?

Reply:
I removed this because of the following line:

checkNotNull(clazz, "response class must not be null");

Reply:
The annotation was on `Object content`, which is nullable?

Reply:
Ah sorry, I was looking at the wrong variable. This will be fixed when I push my other changes.